src/common/hooks/analytics/use-analytics.ts

```diff
@@ -0,0 +1,51 @@
+import { useCurrentNetworkState } from '@store/network/networks.hooks';
+import { useLocation } from 'react-router-dom';
+import { IS_TEST_ENV } from '@common/constants';
+import { EventParams, PageParams } from '@segment/analytics-next/dist/pkg/core/arguments-resolver';
+import { analytics } from '@common/segment-init';
+import { logger } from '@common/logger';
+
+const IGNORED_PATH_REGEXPS = [/^\/$/];
```

Comment:
Do we ignore anything? If so, can we remove it entirely?

Reply:
Yes we just ignore the home '/' which means nothing.